riotfile.py

```diff
@@ -129,7 +129,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             ],
         ),
         Venv(
-            pys=["3.9"],
+            pys=["3.10"],
```

Comment:
We'll probably need to bump this to py310 as well: https://github.com/DataDog/dd-trace-py/blob/1.x/.circleci/config.yml#L294

Reply:
Good catch, thank you!